app/uk/gov/hmrc/bindingtariffclassification/model/NewCaseRequest.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.bindingtariffclassification.model
+
+import java.time.ZonedDateTime
+
+case class NewCaseRequest
+(
+  application: Application,
+  attachments: Seq[Attachment] = Seq.empty
+) {
+  def toCase(reference: String): Case = {
+    val now = ZonedDateTime.now()
+    Case(reference, CaseStatus.NEW, now, now, None, None, None, None, application, None, attachments)
```

Comment:
Maybe something more elegant

Case (reference = reference , status = CaseStatus.NEW, application = application, attachments = attachments )

Still thinking this is mapper though ;)
mapping from caseRequest (or applicationRequest) to Case